docs/storybook-react/.storybook/theme.ts

```diff
@@ -4,7 +4,6 @@ export default create({
   base: 'light',
   brandTitle: 'MetaMask Design System React UI',
   brandUrl: 'https://metamask.io/',
-  brandImage:
-    '../src/stories/assets/logo.svg',
+  brandImage: '../stories/assets/logo.svg',
```

Comment:
brand image still works

<img width="524" alt="Screenshot 2024-10-16 at 12 44 30 PM" src="https://github.com/user-attachments/assets/3d459c71-5142-4b36-aa77-099a4fd79d45">

Reply:
btw that issue you had with image was likely just a hard refresh issue